test/tracer/tracer.c

```diff
@@ -169,15 +226,17 @@ int main( int argc, char** argv ) {
                 : "t3", "t4" );
 
   // trace some threads
-#if 0
+#if 1
+  printf( "Thread test starting\n" );
   rev_pthread_t tid1, tid2;
-  rev_pthread_create(&tid1, NULL, (void *)thread1, NULL);
-  rev_pthread_create(&tid2, NULL, (void *)thread2, NULL);
-  rev_pthread_join(tid1);
-  rev_pthread_join(tid2);
+  rev_pthread_create( &tid1, NULL, (void*) thread1, NULL );
+  rev_pthread_create( &tid2, NULL, (void*) thread2, NULL );
+  rev_pthread_join( tid1 );
+  rev_pthread_join( tid2 );
```

Comment:
I just ran `.githooks/pre-commit --reformat` on `rev/devel`, and saw no changes.

I think the MacOS version of `clang-format` you have, and the Ubuntu version of `clang-format` I have, have slight differences in how they handle spaces. So I see space formatting alternating back and forth sometimes.

Fortunately `pre-commit` only runs `clang-format` on files that a commit will change, unless you give it the `--reformat` flag.

Unfortunately `clang-format` does not allow the `.clang-format` file to specify a specific or even a minimum version of `clang-format` that its options should be interpreted as following, so it's hard to write `.clang-format` files which will work the same across different versions. There's nothing in `clang-format` like Perl's `require 5.6.1;`.